content/en/docs/ambient/overview/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Overview
+description: An overview of Istio's ambient dataplane mode.
+weight: 1
+owner: istio/wg-docs-maintainers-english
+test: no
+---
+
+In **ambient mode**, Istio implements its [features](/docs/concepts) using a per-node Layer 4 (L4) proxy, and optionally a per-namespace Layer 7 (L7) proxy. The mesh created when Istio is installed in ambient mode can be referred to as an ambient mesh.
+
+Since workload pods no longer require proxies running in sidecars in order to participate in the mesh, Istio in ambient mode is often informally referred to as "sidecar-less" mesh.
+
+This layered approach allows you to adopt Istio in a more incremental fashion, smoothly transitioning from no mesh, to the secure overlay, to full L7 processing — on a per-namespace basis, as needed. Furthermore, workloads running in different ambient configurations, or with sidecars, interoperate seamlessly, allowing users to mix and match capabilities based on the particular needs as they change over time.
+
+## How it works
+
+Ambient mode splits Istio’s functionality into two distinct layers. At the base, the **ztunnel** secure overlay handles routing and zero trust security for traffic. Above that, when needed, users can enable L7 **waypoint proxies** to get access to the full range of Istio features. Waypoint proxies, while heavier than the ztunnel overlay alone, still run as an ambient component of the infrastructure, requiring no modifications to application pods.
+
+{{< tip >}}
+Pods/workloads using sidecar proxies can co-exist within the same mesh as pods that operate in ambient mode. Mesh pods that use sidecar proxies can also interoperate with pods in the same Istio mesh that are running in ambient mode. The term "ambient mesh" refers to an Istio mesh that has a superset of the capabilities and hence can support mesh pods that use either type of proxying.
```

Comment:
```suggestion
Pods/workloads using sidecar {{< gloss > }}data plane{{< /gloss > }} modes can co-exist within the same mesh as pods that use ambient data plane mode. The term "ambient mesh" refers to an Istio mesh that was installed with ambient mode enabled, and so can support mesh pods that use either type of data plane.
```

Reply:
(General rule; only use the glossary for the first instance of a term on a page.)

Reply:
I lost track of where we had mentioned it before yea